packages/clerk-js/src/ui/utils/colors/modern.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * Modern CSS-based color manipulation utilities
+ * Uses color-mix() and relative color syntax when supported
+ */
+
+import { cssSupports } from '../cssSupports';
+import { hasModernColorSupport as hasModernColorSupportCached } from './cache';
+
+/**
+ * Modern CSS-based color manipulation utilities
+ * Uses color-mix() and relative color syntax when supported
+ */
+export const modernColors = {
+  /**
+   * Lightens a color by a percentage using modern CSS
```

Comment:
```suggestion
   * Lightens a color by a percentage
```

since we're in `modern.ts` we probably don't need to keep repeating that we're using modern CSS